stack.yaml

```diff
@@ -1,13 +1,19 @@
 resolver: lts-17.6
+
+packages:
+  - "."
+
+extra-deps:
+  # purescript 0.14.3
+  - github: purescript/purescript
+    commit: b4c90a8bb6a1059f271f4211687192370fda81bc
```

Comment:
Why not use the same commit that `v0.14.3` was tagged against, namely, `c1f9ccc6dc6771ed75cb45268c3eae89f1905f8c`?

Reply:
Same as with Pursuit, I think we should get the compiler uploaded to Hackage first and then just refer to it here as `purescript-0.14.3`